### PR TITLE
Do cgroup2 compatible fix for memtune test

### PR DESCRIPTION
**Pull request by kylazhang**
Signed-off-by: Kyla Zhang <weizhan@redhat.com>

### CODE REVIEW

**Review by kamilvarga — APPROVED** [review]
LGTM

**Review by dzhengfy — COMMENTED** [review]
Please paste the test result before and after your fix. The codes LGTM.

File: libvirt/tests/src/virsh_cmd/domain/virsh_memtune.py [hunk 1]
```diff
@@ -4,21 +4,16 @@
 
 from virttest import virsh
 from virttest import utils_libvirtd
+from virttest import libvirt_cgroup
```
Comment (Yingshun) [inline libvirt/tests/src/virsh_cmd/domain/virsh_memtune.py:7]: We'd better have imports in alphabetical order, could you please update this line?

Comment (kylazhang) [reply libvirt/tests/src/virsh_cmd/domain/virsh_memtune.py:6]: Thanks, updated

**Review by Yingshun — APPROVED** [review]

**Comment by kylazhang** [comment]
It depends on https://github.com/avocado-framework/avocado-vt/pull/3081
```
 (01/12) type_specific.io-github-autotest-libvirt.virsh.memtune.positive_test.normal_1: PASS (40.29 s)
 (02/12) type_specific.io-github-autotest-libvirt.virsh.memtune.positive_test.normal_2: PASS (30.85 s)
 (03/12) type_specific.io-github-autotest-libvirt.virsh.memtune.positive_test.normal_3: PASS (40.70 s)
 (04/12) type_specific.io-github-autotest-libvirt.virsh.memtune.positive_test.normal_4: PASS (38.42 s)
 (05/12) type_specific.io-github-autotest-libvirt.virsh.memtune.positive_test.normal_5: PASS (41.19 s)
 (06/12) type_specific.io-github-autotest-libvirt.virsh.memtune.positive_test.normal_6: PASS (645.93 s)
 (07/12) type_specific.io-github-autotest-libvirt.virsh.memtune.positive_test.normal_7: ERROR: Failures occurred while postprocess:\n\nPostprocess: Leaked clusters were noticed during image check. No data integrity problem was found though. (/var/lib/avocado/data/avocado-vt/images/jeos-27-x86_64.qcow2) (104.37 s)
 (08/12) type_specific.io-github-autotest-libvirt.virsh.memtune.positive_test.step_increment: PASS (40.42 s)
 (09/12) type_specific.io-github-autotest-libvirt.virsh.memtune.negative_test.invalid_1: FAIL: Expect should fail with one of ["Unable to write to '.*/memory.limit_in_bytes': Device or resource busy"], but succeeded: \n\n (131.18 s)
 (10/12) type_specific.io-github-autotest-libvirt.virsh.memtune.negative_test.invalid_2: PASS (43.42 s)
 (11/12) type_specific.io-github-autotest-libvirt.virsh.memtune.negative_test.invalid_3: PASS (32.28 s)
 (12/12) type_specific.io-github-autotest-libvirt.virsh.memtune.negative_test.invalid_4: PASS (32.47 s)
RESULTS    : PASS 10 | ERROR 1 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1224.21 s

1 failure rerun pass
(1/1) type_specific.io-github-autotest-libvirt.virsh.memtune.positive_test.normal_7: PASS (17.69 s) 
should related to last result which broke the image

Another one fail for negative is expected fail
```

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**